docs/pages/getting-started/digitalocean.mdx

```diff
@@ -0,0 +1,119 @@
+<Figure align="left" bordered caption="Create a droplet">
+  ![Create a droplet](../../img/quickstart/digitalocean/create-droplet.png)
+</Figure>
+
+It will take a few minutes before our newly created Teleport droplet is ready. Once the droplet is ready, configure your FQDN with the public IP address of the droplet as an IP address for the `A` record of your domain name. 
+For example, refer to the image below; we use the domain name `teleporters.dev`. The resulting domain we are using as an FQDN is `dodemo.teleporters.dev`, pointing to our Teleport droplet's public IP `192.168.200.200`.
+<Figure align="left" bordered caption="Configure DNS">
+  ![Configure DNS](../../img/quickstart/digitalocean/fqdn.png)
+</Figure>
+
+## Step 2/3. Configure Teleport
+When you are ready with your FQDN, SSH to your droplet. In your first login, a wizard will guide you through the initial Teleport setup.
+
+```
+$ ssh root@teleport_host
+*********************************************************
+**              Configuring Teleport                   **
+*********************************************************
+___
+• Enter Teleport cluster name (FQDN): dodemo.teleporters.dev (replace with your FQDN)
+• Enter your email address to retrieve TLS certificate from Lets Encrypt: dodemo@teleporters.dev (replace with your email address)
+• Enter a username for the initial Teleport user: tadmin
+
+Initializing...
+[+] Generating new host UUID...
+[+] Updating cluster networking configuration...
+[+] Generating user and host certificate authority...
+[+] Enabling RBAC in OSS Teleport. Migrating users, roles and trusted clusters...
+[+] Starting Auth and Proxy services...
+[+] Final Checks...
+
+***************************************************************************
+**                             ---
+**    Teleport is configured and user tadmin has been created
+**    but requires a password. Open the URL link below to complete the
+**    setup. The link is valid for 1h:
+**
+**    https://dodemo.teleporters.dev:443/web/invite/<invite-token>
+**
+**                             ---
+**                     HAPPY TELEPORTING :)
+***************************************************************************
+```
+
+Copy the URL link printed in this step:
+`https://dodemo.teleporters.dev:443/web/invite/<invite-token>`
+
+This link opens up Teleport Web UI, where you will need to set a password and configure two-factor authentication to complete the user setup process.
+
+## Step 3/3. Complete user setup and log in to Teleport UI
+Open the link copied in the previous step in the browser to complete the setup process. When the web page is ready:
+1. Scan the QR code with your two-factor authentication app (e.g., Google Authenticator)
```

Comment:
If using Teleport 8+ we can use WebAuth and provide the option for a hardware or OTP for 2nd factor? Maybe an addition for the next version.